AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -264,8 +264,8 @@ radeberger.com##.cookieabfrage
 montiemare.de#$##cookieNotice { display: none !important; }
 montiemare.de#$#.modal-open { overflow: visible !important; }
 montiemare.de#$##cookieNotice + .modal-backdrop { display: none! important; }
-zdnet.com#$##onetrust-consent-sdk { display: none !important; }
-zdnet.com#$#button#ot-sdk-btn { display: none !important; }
+cnet.com,zdnet.com#$##onetrust-consent-sdk { display: none !important; }
+cnet.com,zdnet.com#$#button#ot-sdk-btn { display: none !important; }
```

Comment:
If I'm not wrong, these rules hide cookie notice, so I think that we should add them only for platforms which can update/download new JavaScript rules.
For example, extensions for Firefox and Opera can't update/download new JavaScript rules (due to Mozilla/Opera policy), so this cookie notice will be hidden and some of website functionality will be broken/not work.

Reply:
It looks like that we don't need these css rules anymore (because your new js rule hides cookie-popup instantly), so I think that we should leave only rule `cnet.com,zdnet.com###ot-sdk-btn` for hiding "manage cookies" button.

Reply:
Ok, I think that it will be fine.